ddpui/core/charts/chart_validator.py

```
@@ -16,7 +16,7 @@ class ChartValidator:
     """Validates chart configurations based on chart type"""
 
     # Valid chart types
-    VALID_CHART_TYPES = ["bar", "pie", "line", "number", "map", "table"]
+    VALID_CHART_TYPES = ["bar", "pie", "line", "number", "map", "table", "pivot_table"]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)